apps/gp2-server/test/controllers/working-group.controller.test.ts

```diff
@@ -32,10 +32,37 @@ describe('Working Group controller', () => {
         total: 0,
         items: [],
       });
-      const result = await workingGroupController.fetch();
+      const result = await workingGroupController.fetch('11');
 
       expect(result).toEqual({ items: [], total: 0 });
     });
+
+    test('Should removes the resource if the user is not a member of the working group', async () => {
```

Comment:
I think I would like to see another test (and it could be the only test really) which should show that a user needs to be the same user of that in the WG members list. In here the list is empty so its not exactly clear

Reply:
done